rest_api/pipeline/pipeline_empty.haystack-pipeline.yml

```diff
@@ -1,5 +1,5 @@
 # Dummy pipeline, used when the CI needs to load the REST API to extract the OpenAPI specs. DO NOT USE.
-version: 'unstable'
+version: 'master'
```

Comment:
Good point. I originally excluded `latest` because it often means "latest released version", which in this case might be misleading. `current` and `any` aren't bad but I've never seen them used, probably just because they're vague... And leaving out the version altogether is an option, but it might hide the fact that you _can_ specify the version if the versionless YAML become more popular (which they might).

On the other hand, I'm not sure tying the name to github is an issue. Whoever needs to use this label is very likely to be actually installing from master. We could find more generic ones like `source`, `git` or something like this, to avoid referencing the branch name? Even though those are rarely used in the wild as well.

Or maybe `version: ignore` to be even more upfront? 

I think I could go for `ignore`, `source` or `any`, but let me know what do you think.

Reply:
I like `ignore` the most. This gives the best hint about what it means.

Reply:
The json schema file will still be called `haystack-pipelines-master.schema.json` just because `haystack-pipelines-ignore.schema.json` sounded odd :smile: But it has no impact on the content of the `version` field in YAML. Renamed all the rest :+1: